package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "learn-react",
+  "version": "1.0.0",
+  "description": "Learn react",
+  "main": "index.js",
+  "scripts": {
+    "start": "cross-env NODE_ENV=development webpack-dev-server --open",
+    "build:prod": "cross-env NODE_ENV=production webpack",
+    "build:dev": "cross-env NODE_ENV=development webpack",
```

Comment:
For what reason we need build.dev command?

Reply:
1)	I want to see files and use source-map for easy debugging in build mode
2)	I don’t want to minify CSS and Html